torchaudio/models/wav2vec2/pretrained.py

```diff
@@ -160,6 +161,82 @@ def _get_labels():
 [`Source <https://github.com/pytorch/fairseq/tree/main/examples/hubert#pre-trained-and-fine-tuned-asr-models>`__]
 """
 
+HUBERT_LARGE = Wav2Vec2PretrainedModelBundle(
+    'hubert_fairseq_large_ll60k.pth',
+    {
+        'extractor_mode': 'layer_norm',
+        'extractor_conv_layer_config': [
+            (512, 10, 5),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 2, 2),
+            (512, 2, 2),
+        ],
+        'extractor_conv_bias': False,
+        'encoder_embed_dim': 1024,
+        'encoder_projection_dropout': 0.0,
+        'encoder_pos_conv_kernel': 128,
+        'encoder_pos_conv_groups': 16,
+        'encoder_num_layers': 24,
+        'encoder_num_heads': 16,
+        'encoder_attention_dropout': 0.0,
+        'encoder_ff_interm_features': 4096,
+        'encoder_ff_interm_dropout': 0.0,
+        'encoder_dropout': 0.0,
+        'encoder_layer_norm_first': True,
+        'encoder_layer_drop': 0.0,
+        'aux_num_out': None,
+    },
+    _labels=None,
+)
+HUBERT_LARGE.__doc__ = """HuBERT model with "Large" configuration.
+
+Trained on 60,000 hours of *Libri-Light* [:footcite:`librilight`] dataset.
```

Comment:
`HUBERT_BASE` includes `Not fine-tuned.` in the [description](https://github.com/pytorch/audio/pull/1824/files#diff-f8152dc5c8f7b34fdd59cc08f75744d400f06235470b88f3b6076d3523db97ebR158), so I think we should add it here for consistency as well. Same goes for `HUBERT_XLARGE`